object.c

```diff
@@ -106,13 +106,17 @@ rb_obj_setup(VALUE obj, VALUE klass, VALUE type)
     return obj;
 }
 
-/**
- *  call-seq:
- *     obj === other   -> true or false
+/*
+ * call-seq:
+ *   nil === other -> true or false
+ *
+ * Returns +true+ or +false+.
+ *
+ * Like Object#== if +object+ is an instance of Object
```

Comment:
Nit: I think it's more grammatical to have a comma here, WDYT?

```suggestion
 * Like Object#==, if +object+ is an instance of Object
```